sstemplates-core/src/main/java/com/carbonfive/sstemplates/SsTemplateContext.java

```diff
@@ -3,89 +3,100 @@
 import java.io.*;
 import java.lang.reflect.*;
 import java.util.*;
-import javax.servlet.jsp.el.*;
+
+import de.odysseus.el.util.SimpleResolver;
```

Comment:
I was a bit confused with the library that is only used by a few other open source outdated projects but once i noticed that `javax.el` itself is supported by the same `odysseus` company i got rid of my doubts